cli/src/sbt-test/integration/cli/test

```diff
@@ -0,0 +1,29 @@
+> compile
+
+# Print used version
+> runCli --version
+
+# -- Link and check if outpath exists
+> runCli --outpath target/out1 -v -v --main Main$
+$ exists target/out1
+$ exec target/out1
```

Comment:
I think I've made a mistake here, on Windows the fact the file can be treated as executable comes only from its extension. On Unix we don't have such a limitation, so we probably can name it `target/out1.exe` to allow for running on both Windows and Unix. If that will not help we can remove this check - I just wanted to make sure that created file can be run after linking

Reply:
On my local machine .exe is not helping unfortunately (I can run the file manually though, by pausing and running from command line). I think we have to remove it